deployments/dlab/image/environment.yml

```diff
@@ -0,0 +1,114 @@
+name: dlab-sp23
```

Comment:
I'd just call this `dlab` since git will do versioning for you, and since the image will probably be perpetually updated. But appending `-sp23` won't do any harm either.

I'd also do a diff between this file and `deployments/stat159/image/environment.yml` and pull of their updates before this merges.